x-pack/plugins/lists/common/schemas/types/entry_nested.mock.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FIELD, NESTED } from '../../constants.mock';
+
+import { EntryNested } from './entry_nested';
+import { getEntryMatchMock } from './entry_match.mock';
+import { getEntryMatchAnyMock } from './entry_match_any.mock';
+
+export const getEntryNestedMock = (): EntryNested => ({
+  entries: [{ ...getEntryMatchMock() }, { ...getEntryMatchAnyMock() }],
```

Comment:
Nit: there's a lot of these all over the place. I prefer this form if you're adding/updating properties, but if it's just the object returned by the function, it's more legible to just use that value until you need this syntax.

Reply:
👍  Will update in a follow up!

Reply:
Removed in https://github.com/elastic/kibana/pull/73865